cloud/aws/cli/main.py

```diff
@@ -25,21 +32,34 @@ def print_help(self) -> None:
 
   -e, --echo                     Echo executed commands before running.
   -h [STRING], --help[=STRING]   Show core or per-task help and exit.
-  -V, --version                  Show version and exit.      
+  -V, --version                  Show version and exit.
+
+Plugins:
 """
         )
+        active = common.active_plugins()
+        plugin_entry = lambda name: f"[{'x' if name in active else ' '}] {name}"
+        self.print_columns([(plugin_entry(p[0]), p[1].__doc__) for p in list_plugins()])
```

Comment:
With just this tiny tweak, this PR now also solves [35748](https://app.shortcut.com/tenzir/story/35748/vast-cloud-plugin-level-doc)

Reply:
LGTM